dragon/runtime_docs.rb

```diff
@@ -278,11 +278,11 @@ def tick args
 - ~:down_or_held~
 - ~:up~
 ** ~args.outputs~
-~args.outputs.PROPERTY~ is how you render to the screen.
+~args.outputs.PROPERTY~ is how you render to the screen and play sounds. Rendering to the screen happens in the following order: ~solids < sprites < primitives < labels < lines < borders < debug~ and ~static_*~ are drawn above their non-static counterparts.
```

Comment:
Suggest “static outputs” instead of “static_*”

Reply:
Agreed, `static_*` is a pretty "inside" term in my opinion, most people just getting into DR probably won't know what that's supposed to mean.

Maybe even do a small paragraph about the difference between static and non-static primitives? (If it doesn't exist already (which I don't think it does?))

Reply:
Yes, this is a fair comment. I'll whip something up.

Reply:
Please have another look and see if the new text makes sense.